assets/js/map.js

```diff
@@ -0,0 +1,38 @@
+
+var mapOptions = {
+    center: [44.8, -2.46],
+    zoom: 3
+}
+
+// instantiating the map with user defined mapOptions passed as parameter
+var map = new L.map("map", mapOptions);
+
+// instantiating the TileLayer Class which displays the desired map UI (passed as a String parameter)
+var layer = new L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png');
+
+// add our custom layer to the map object created above
+map.addLayer(layer);
+
+// const DATA = {{ site.data | jsonify }};
+
+console.log(DATA);
```

Comment:
Generally, it's not a good practice to leave debug statements in production code.

Reply:
Ah yess! will take care of it from next time onwards 👍🏻